force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +182,10 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
-
+        Integer randomNumber = (Integer)(Math.random() * 50) + 50;
+        randomNumber = math.round(randomNumber);
         // Return the random integer
-        return null; // Replace null with the variable you used to store the result
+        return randomNumber; // Replace null with the variable you used to store the result
     }
```

Comment:
This is great. Another way to do this is to just return from Math.round and just pass it the results of Math.ranndom()

```
  public static Integer generateRandomNumber() {
        // Use Math class to generate number between 50 and 100
        return Math.round(Math.random() * 50) + 50;
    }
```